src/store.js

```diff
@@ -1,10 +1,11 @@
-import { configureStore } from "@reduxjs/toolkit";
```

Comment:
Kindly enable eslint checks as it helps to enforce consistency and help validate errors in your code to make it look clean.

Reply:
thanks! its fixed now!